_targets.R

```diff
@@ -22,13 +24,15 @@ list(
 
   # TODO: PULL SITE DATA HERE
   tar_map(
-    values = tibble(state_abb = states),
+    values = tibble(state_abb = states)%>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
     tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == state_abb)),
     tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
     # Insert step for tallying data here
-    tar_target(count, tally(nwis_data)),
+    tar_target(tally, tally_site_obs(nwis_data)),
     # Insert step for plotting data here
-    tar_target(fig, plot(count))
+    tar_target(timeseries_png, plot_site_data(state_plot_files, nwis_data, parameter)),
+    names = state_abb
```

Comment:
I know the course instructions say to add this as the final argument, but I thought I'd make a note here that, from a readability standpoint, I like to have this `names` specification up above right after `values` is defined, so that someone encountering the pipeline can see how the target branches will be named before the targets are defined. No need to make a change, though.